package.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "infusion-electron",
+    "description": "Infusion component wrappers for Atom Electron",
+    "version": "0.1.0",
+    "main": "index.js",
+    "author": "Colin Clark",
+    "license": "BSD-3-Clause",
+    "repository": "git://github.com/colinbdclark/infusion-electron.git",
+    "bugs": "http://github.com/colinbdclark/infusion-electron/issues",
+    "homepage": "http://github.com/colinbdclark/infusion-electron",
+    "readmeFilename": "README.md",
+    "dependencies": {
+        "infusion": "2.0.0-dev.20151007T134312Z.8c33cd4",
+        "dedupe-infusion": "1.0.0",
+        "electron-prebuilt": "0.33.7"
+    },
+    "devDependencies": {
+        "grunt-contrib-jshint": "0.9.0",
+        "grunt": "0.4.5"
+    },
+    "scripts": {
+        "dedupe-infusion": "node -e \"require('dedupe-infusion')();\""
```

Comment:
If it's worth exposing this, it's probably worth wrapping the full install process in a task of its own as per https://github.com/fluid-project/first-discovery-server/pull/2/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R12 - or if the direct user of the module is unlikely to run it at this level, it can go in the README